src/Kkb/Ding2IntegrationBundle/Service/Ding2Service.php

```diff
@@ -116,118 +117,109 @@ protected function updateOpeningHours() {
     foreach ($slides as $slide) {
       $options = $slide->getOptions();
 
+      // Get the Drupal Node ID we require to do anything.
+      if (empty($nid = $options['feed']['library'])) {
+        continue;
+      }
```

Comment:
This seems odd to me.

We check for `$options['feed']['library']` here. A few lines down we check for `$options['feed']`. On L134 we check for `$options['feed']['library']` again.

Also I am not too keen on variable assignments in a condition.

Reply:
👍 I accidentially duplicated the check during a late refactor, I'll update the commit with a single check instead of 2 :)